code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -97,6 +97,17 @@
 				if(T.canBuildPlating(S) == BUILD_SUCCESS)
 					qdel(L)
 
+/obj/item/stack/tile/wood/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	if(iswrench(W))
+		if(use(4))
+			playsound(get_turf(src), 'sound/items/Ratchet.ogg', 50, 1)
+			drop_stack(/obj/item/stack/sheet/wood, get_turf(user), 1, user)
+		else
+			to_chat(user, "<span class='warning'>You need at least 4 [src] to get a wooden plank back!</span>")
```

Comment:
Then if the name's already plural it'll look like floor tiless.

Now that I think of it stack code with names is fucking awful.